contrib/test_jenkins.sh

```diff
@@ -1119,6 +1131,25 @@ run_asan_check() {
 	run_gtest "default"
 }
 
+run_valgrind_check() {
+	if [[ $(uname -m) =~ "aarch" ]] || [[ $(uname -m) =~ "ppc" ]]
+	then
+		echo "==== Skip valgrind tests ===="
+		return
+	fi
+
+	# Load newer valgrind if naative is older than 3.10
+	if ! (echo "valgrind-3.10.0"; valgrind --version) | sort -CV
```

Comment:
would `if ! { echo "valgrind-3.10.0"; valgrind --version; } | sort -CV; then ...` work, to avoid sub shell?

Reply:
Seems like it works, but why is that better?